xtask/src/commands/changeset/mod.rs

```diff
@@ -66,15 +66,29 @@ struct TemplateContext {
 
 const REPO_WITH_OWNER: &'static str = "apollographql/router";
 
-const EXAMPLE_TEMPLATE: &'static str = "### {title} {{ for issue in issues -}}
-([Issue #{issue.number}]({issue.url})){{ if not @last }}, {{ endif }}
-{{- endfor }}
+const EXAMPLE_TEMPLATE: &'static str = "### { title }
+{{- if issues -}}
+  {{- if issues }} {{ endif -}}
```

Comment:
```suggestion
```
shouldn't this be removed?

Reply:
Good eye and question, but no and hard to comment why it exists. It actually allows me to use a nice formatted structure for this template, while ensuring that there will be a space **preserved** there at all times.